content/cpp/concepts/math-functions/terms/fmin/fmin.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: 'fmin()'
+Description: 'Returns the smaller of two arguments.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`fmin()`** function returns the smaller of two arguments.
+
+## Syntax
+
+```pseudo
+fmin(x, y)
+```
+
+The parameters `x` and `y` are usually a numeric [data type](https://www.codecademy.com/resources/docs/cpp/data-types) ranging from `-INFINITY` (negative) to `INFINITY` (positive). If one of the parameters is `NAN`, then the other parameter is returned.
```

Comment:
- Similar to the open PR for `fmax()` function, can we change  to to "negative `INFINITY` to positive `INFINITY`."
- Change `NAN` to `NaN`